server/setup.cfg

```
@@ -9,3 +9,9 @@ ignore = E203, E266, E501, W503
 include_trailing_comma = True
 line_length = 100
 multi_line_output = 3
+
+[mypy]
+plugins = pydantic.mypy
+follow_imports = silent
+ignore_missing_imports = True
+allow_redefinition = True
```

```suggestion
allow_redefinition = True
check_untyped_defs = True
```
pratique pour avoir du typecheck même sans signature (par exemple sur les tests)

---

https://github.com/ToucanToco/weaverbird/pull/670/commits/a1c81bfa3012c195f55c66eaaa0f1e9b551c91ad